modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/ColorSwatchItem.ts

```diff
@@ -53,3 +53,15 @@ export const renderColorSwatchItem = (spec: Menu.FancyMenuItem, backstage: UiFac
     ]
   };
 };
+
+const getcolorItems = (spec: Menu.FancyMenuItem, backstage: UiFactoryBackstage): Menu.ChoiceMenuItemSpec[] => {
+  if (Type.isNonNullable(spec.initData)) {
+    if (spec.initData.colorselection) {
+      return spec.initData.colorselection.concat(ColorSwatch.getAdditionalColors(false));
+    } else {
+      return ColorSwatch.getColors(backstage.colorinput.getColors(), false);
```

Comment:
Why is this different? This should either be using the provided colors configuration, or always falling back to the same default. As such, I'd suggest rewriting this function to something like this:
```
const getColorItems = (spec: Menu.FancyMenuItem, backstage: UiFactoryBackstage): Menu.ChoiceMenuItemSpec[] =>
  Optional.from(spec.initData.colors)
    .filter(Type.isArray)
    .map((colors) => colors.concat(ColorSwatch.getAdditionalColors(false)))
    .getOrThunk(() => ColorSwatch.getColors(backstage.colorinput.getColors(), backstage.colorinput.hasCustomColors()));
```